src/components/molecules/NavList.tsx

```diff
@@ -31,34 +26,59 @@ export function NavList() {
     items.map(item => ({
       label: enumMappings[item].label,
       filter: item,
-      path: enumMappings[item].url, 
+      path: enumMappings[item].url,
     }));
 
-  const subheaderLinkItems: SubheaderLinksProps[] = [
-    { items: roles, navigate },
-    { items: genres, navigate },
-  ]
+  const combinedLinks: ListButtonProps[] = [
+    ...navLinks.map(item => ({
+      label: item.label,
+      path: item.route,
+    })),
+    ...generateButtons(roles),
+    ...generateButtons(genres),
+  ];
+
+  const secondNavLinkOrder = [
+    'Animation',
+    'Documentary',
+    'Director of Photography',
+    'Director & Producer',
+    'Series',
+  ];
+
+  const secondNavLinks = combinedLinks
+    .filter(item => secondNavLinkOrder.includes(item.label))
+    .sort((a, b) => secondNavLinkOrder.indexOf(a.label) - secondNavLinkOrder.indexOf(b.label));
 
   return (
-    <List
-      sx={{ width: '100%', height: '100%', maxWidth: 360, ...NavStyle }}
-      component="nav"
-    >
-      {navLinks.map(item => (
-        <NavLinks
-          key={item.route}
-          route={item.route}
-          label={item.label}
-          onClick={() => navigate(item.route)}
-        />
-      ))}
+    <div>
+      <List
+        sx={{ width: '100%', paddingBottom: '20px', maxWidth: 360, ...NavStyle }}
+        component="nav"
+      >
+        {navLinks.map((item, index) => (
+          <NavLinks
+            key={index}
+            route={item.route}
+            label={item.label}
+            onClick={() => navigate(item.route)}
+          />
+        ))}
+      </List>
 
-      {subheaderLinkItems.map(item => (
-        <SubheaderLinks
-          links={generateButtons(item.items)}
-          navigate={navigate}
-        />
-      ))}
-    </List>
+      <List
+        sx={{ width: '100%', height: '100%', maxWidth: 360, ...NavStyle }}
+        component="nav"
+      >
+        {secondNavLinks.map((item, index) => (
+          <NavLinks
+            key={index}
+            route={item.path}
```

Comment:
❓ Why give a `route` and an `onClick` that navigates?

Reply:
`route` is not necessary and I just didn't think about it when I sent up this PR. 
We (I) can just utilize `useNavigate() here.